backend/schema/validate.go

```diff
@@ -464,7 +466,19 @@ func validateVerbMetadata(scopes Scopes, n *Verb) (merr []error) {
 				case *IngressPathLiteral:
 				}
 			}
+		case *MetadataCronJob:
+			if hasCron {
+				merr = append(merr, errorf(md, "only a single cron schedule is allowed per verb"))
```

Comment:
Hmm this should be generalised to the other metadata types rather than special-casing just for cron - maybe using a set of reflected type names to check for duplicates?

Reply:
Can a verb have multiple html ingresses?

Reply:
Not currently.

Reply:
Mmm actually though, that's a good point - most of them can, and this is the exception, so ignore what I said!